kernel/kernel_drivers.go

```diff
@@ -176,6 +176,11 @@ func createModulesSubtree(kMntPts MountPoints, kernelTree, kversion string, comp
 }
 
 func createKernelModulesSymlinks(modsRoot, kMntPt string) error {
+	// FIXME: having a symlink for something not yet mounted is
```

Comment:
This feels incredibly fragile - is this something we need to discuss a proper solution for as a topic for 25.10?

Reply:
This comment is not correct, we do mount the snap and kernel-modules components from the initramfs, see https://github.com/canonical/snapd/blob/master/cmd/snap-bootstrap/cmd_initramfs_mounts.go#L2106